fleetctl/fleetctl.go

```diff
@@ -732,6 +803,64 @@ func lazyCreateUnits(args []string) error {
 	return nil
 }
 
+// matchUnitFiles compares two unitFiles
+// Returns true if the units match, false otherwise.
+func matchUnitFiles(a *unit.UnitFile, b *unit.UnitFile) bool {
+	if a.Hash() == b.Hash() {
+		return true
+	}
+
+	return false
+}
```

Comment:
Putting this in a function makes no sense: the invocation isn't any simpler than the comparison itself; nor does it make the code easier to read I'd say.

It might still be justified if this were encapsulating some internal detail that is very likely to change -- but I don't see that here...